src/colorizer/types.ts

```diff
@@ -110,3 +121,39 @@ export const isThresholdCategorical = (threshold: FeatureThreshold): threshold i
 export const isThresholdNumeric = (threshold: FeatureThreshold): threshold is NumericFeatureThreshold => {
   return threshold.type === ThresholdType.NUMERIC;
 };
+
+export type DrawSettings = {
+  mode: DrawMode;
+  color: Color;
+};
+
+/**
+ * Configuration for the viewer. These are high-level settings
+ * that are not specific to a particular dataset.
+ */
+export type ViewerConfig = {
+  showTrackPath: boolean;
+  showScaleBar: boolean;
+  showTimestamp: boolean;
+  keepRangeBetweenDatasets: boolean;
+  backdropBrightness: number;
+  backdropSaturation: number;
+  objectOpacity: number;
+  outOfRangeDrawSettings: DrawSettings;
+  outlierDrawSettings: DrawSettings;
+};
+
+export const defaultViewerConfig: ViewerConfig = {
+  showTrackPath: true,
+  showScaleBar: true,
+  showTimestamp: true,
+  keepRangeBetweenDatasets: true,
+  /** Opacity, as a number percentage. */
+  backdropBrightness: 100,
+  /** Saturation, as a number percentage. */
+  backdropSaturation: 100,
+  /** Opacity, as a number percentage. */
+  objectOpacity: 100,
```

Comment:
Arguably these can be per-dataset settings, but that's a change for another PR :')